### ISSUE TITLE
Issue #66: Unable to listen to messages with cluster.Consumer - still having the same problem

### ISSUE THREAD

**Issue by mslepak** [bsm/sarama-cluster#67]
Dimitrij, thanks for looking into the issue #66 I posted last Friday. Unfortunately, your changes haven't fixed the problem I am experiencing. I am still not able to receive any published messages. I am on commit 659b67fc3f26f8d7744cdc0ce15758f3c59ca843.

Here is an interesting fact. If I change line 21 in partitioins.go from

	pcm, err := manager.ConsumePartition(topic, partition, info.NextOffset(defaultOffset))

to

	pcm, err := manager.ConsumePartition(topic, partition, sarama.OffsetOldest)

I will then be able to see all messages that I've ever published. This is obviously not the desired behavior, but you see my point...

The other interesting thing is that if I use OffsetNewest on line 21 (which makes more sense that OffsetOldest), then I see NO messages at all.

Thanks again for your help.

**Comment by mslepak** [comment]
Well, after running a bunch of tests, I found what I was doing wrong. Apparently, Kafka 0.9 differs in its behavior from the earlier versions. So, if I am using the default setting (which is the largest offset), then I won't be able to listen to messages published before the consumer starts listening, and that's exactly how my tests were setup. The moment I delayed the producer, I started receiving all my published messages. So, it's all good.
